File: Python/lista.py
class Lista:
    def __init__(self):
        self.lista = []

    def insere(self, numero):
        self.lista.append(numero)

    def retirar(self):
        if self.tamanho() > 0:
            self.lista.pop()
        else:
            print("ERRO, Lista vazia")

    def primeiro(self):
        if self.tamanho() > 0:
            return self.lista[0]
        else:
            print("ERRO, Lista vazia")
            return -1

    def tamanho(self):
        return len(self.lista)

File: Rust/lista.rs
pub struct Lista {
    pub lista: Vec<i32>,
}

impl Lista {
    pub fn new() -> Self {
        Lista { lista: Vec::new() }
    }

    pub fn insere(&mut self, numero: i32) {
        self.lista.push(numero);
    }

    pub fn retirar(&mut self) {
        if self.tamanho() > 0 {
            self.lista.pop();
        } else {
            println!("ERRO, Lista vazia");
        }
    }

    pub fn primeiro(&self) -> i32 {
        if self.tamanho() > 0 {
            self.lista[0]
        } else {
            println!("ERRO, Lista vazia");
            -1
        }
    }

    pub fn tamanho(&self) -> usize {
        self.lista.len()
    }
}
